pkg/controllers/work/apply_controller.go

```diff
@@ -629,6 +672,144 @@ func buildManifestAppliedCondition(err error, action applyAction, observedGenera
 	}
 }
 
+func (r *ApplyWorkReconciler) createConfigMap(ctx context.Context, configMapName string, owner metav1.OwnerReference, manifestHash, lastModifiedConfig string) error {
+	configMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: fleetSystemNamespace,
+		},
+		Data: map[string]string{},
+	}
+	configMap.Data[manifestHashKey] = manifestHash
+	configMap.Data[lastAppliedConfigKey] = lastModifiedConfig
+	// add applied work owner reference. let's check if it works
```

Comment:
Comment doesn't add value.

Reply:
This is a draft PR, was using comments to keep track